workspaces/ui-v2/src/optic-components/shapes/ShapeRenderInterfaces.ts

```diff
@@ -15,12 +15,13 @@ export interface IShapeRenderer {
   jsonType: JsonLike;
   asArray?: IArrayRender;
   asObject?: IObjectRender;
-  value: any;
+  value?: any;
```

Comment:
We aren't actually querying this, and i believe `any` actually include nullable - but I added a `?` to make it more explicit that we can't rely on this value.